counter/cookieManager.js

```diff
@@ -0,0 +1,24 @@
+function setCookie(name, value, attrs) {
+	const attributes = {
+		path: '/',
+		...attrs
+	}
+
+	let newCookie = `${name}=${value}`;
+
+	for(let attr in attributes) {
+		newCookie += `; ${attr}=${attributes[attr]}`; 
+	}
+
+	document.cookie = newCookie;
+}
+
+function getCookie(name) {
```

Comment:
Предлагаю переименовать эту функцию, чтобы было понятно, о чем идёт речь
```suggestion
function getCookieValueByName(name) {
```